mypy/build.py

```diff
@@ -768,7 +767,10 @@ def correct_rel_imp(imp: Union[ImportFrom, ImportAll]) -> str:
                     # if all of the imports are submodules, do the import at a lower
                     # priority.
                     pri = import_priority(imp, PRI_HIGH if not all_are_submodules else PRI_LOW)
-                    res.insert(pos, ((pri, cur_id, imp.line)))
+                    # The imported module goes in after the
```

Comment:
(I was curious why this was an `res.insert` to start with. It looks like it was introduced in the dawn of incremental mode #1292 and was previously a `res.append` before the loop. It was moved after the loop to change the priority, and so I guess `res.insert` was to maintain the previous position)

Reply:
Ah, interesting!